psakefile.ps1

```diff
@@ -4,10 +4,10 @@ properties {
     $solution_name = "MarkdownWebsite"
     $solution_dir = "$base_dir"
 	$solution_path = "$solution_dir\$solution_name.sln"
-	$runtime_id = "win10-x64"
+	$runtime_id = "win-x64"
     $project_dir = "$solution_dir\$solution_name"
     
-    $output_dir = "$project_dir\bin\Debug\netcoreapp3.1"
+    $output_dir = "$project_dir\bin\Debug\netcoreapp3.1\win-x64\publish"
```

Comment:
This breaks the default flow, which just does a clean and build, not a publish, so if you haven't published before it won't be able to execute the last step which is to build the sample website.

You can test this yourself by deleting your build folder, opening a command line at the repository root and running `build.bat`.

Reply:
Removed the `publish` portion of the path and tested from clean, build, and then publish